src/components/enterprise-user-subsidy/UserSubsidy.jsx

```diff
@@ -32,9 +33,9 @@ const UserSubsidy = ({ children }) => {
         hasAccessToPortal = subscriptionLicense?.status === LICENSE_STATUS.ACTIVATED;
       }
 
-      return { hasAccessToPortal, subscriptionLicense };
+      return { hasAccessToPortal, subscriptionLicense, offers };
     },
-    [subscriptionPlan, subscriptionLicense],
+    [subscriptionPlan, subscriptionLicense, offers, enterpriseConfig.uuid],
```

Comment:
Is there ever a chance that `enterpriseConfig` will be `null` or `undefined` here, where this would cause a JS error? Would just using `enterpriseConfig` suffice here?

Reply:
Good question. I think if the enterprise config is not there we probably have bigger issues? I was thinking of the possibility that I navigate from one enterprise to another (which isn't *easy* right now, but is totally possible). In that case, I definitely want to check if the uuid has changed.

Reply:
I think we'd be safe to just to check if the `enterpriseConfig` object itself has changed in this case.

Reply:
My understanding is that if I use `enterpriseConfig`,  if other things in `enterpriseConfig` are changing that we don't care about here, this will update. I don't care about anything in `enterpriseConfig` except it's uuid.  But maybe that's over optimizing.

Reply:
If the uuid is changing in `enterpriseConfig`, I would likely expect all the values in the object to be changing since it's a different enterprise :)